tests/vmi_iothreads_test.go

```diff
@@ -216,7 +216,7 @@ var _ = Describe("[sig-compute]IOThreads", decorators.SigCompute, func() {
 			Entry("[test_id:3095] for three CPUs", 3, 6),
 			// there's only 6 threads expected because there's 6 total disks, even
 			// though the limit would have supported 8.
-			Entry("[test_id:3096]for four CPUs", 4, 6),
+			Entry("[test_id:3096]for four CPUs", 4, 7),
```

Comment:
Why do we need to change from 6 to 7?

Reply:
Same reason as this comment.
https://github.com/kubevirt/kubevirt/pull/12516/files/58a731f3dc057f525d5d86e907c4d3f60d74e03c#r1734287174